tests/counterpoll_test.py

```diff
@@ -44,6 +60,14 @@ def test_show(self):
         print(result.output)
         assert result.output == expected_counterpoll_show
 
+    @mock.patch('counterpoll.main.device_info.get_platform_info')
+    def test_show_dpu(self, mock_get_platform_info):
+        mock_get_platform_info.return_value = {'switch_type': 'dpu'}
+        runner = CliRunner()
+        result = runner.invoke(counterpoll.cli.commands["show"], [])
+        print(result.output)
```

Comment:
Is this print required here? Looks like debug information that can be removed

Reply:
removed